### PR TITLE
Use dvm_download in install.sh

### PR DESCRIPTION
**Pull request by rgbkrk**
* Install `dvm` during install.sh run
* For install, rely on `dvm_download` to get both `dvm.sh` and `dvm-helper`.

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -13,10 +71,15 @@ if [ ! -d "$DVM_DIR" ]; then
 fi
 
 echo "Downloading dvm.sh..."
-curl -s -o $DVM_DIR/dvm.sh https://raw.githubusercontent.com/getcarina/dvm/master/dvm.sh
+dvm_download -L -C - --progress-bar https://raw.githubusercontent.com/getcarina/dvm/master/dvm.sh -o $DVM_DIR/dvm.sh
+
+if [ ! -f "${DVM_DIR}/dvm-helper" ]; then
```
Comment (carolynvs) [inline install.sh]: We are unconditionally overwriting dvm.sh but not dvm-helper. Can we remove the if check for dvm-helper and just always ensure that they have the latest when running the install?

Comment (rgbkrk) [inline install.sh]: That seems ok.

Comment (rgbkrk) [inline install.sh]: Actually, that seems more than ok since they should match.

**Comment by carolynvs** [comment]
Merging for now and I'll circle back later to fix the comment.

**Pull request merged**